web/app.js

```diff
@@ -699,7 +703,10 @@ const PDFViewerApplication = {
   },
 
   get supportsPrinting() {
-    return PDFPrintServiceFactory.instance.supportsPrinting;
+    return (
+      !this.externalServices.isMobile &&
+      PDFPrintServiceFactory.instance.supportsPrinting
+    );
```

Comment:
Why isn't it sufficient to simply change this getter as follows, rather than implementing a bunch of other code?
```suggestion
    return
      (typeof PDFJSDev === "undefined" || !PDFJSDev.test("GECKOVIEW")) &&
      PDFPrintServiceFactory.instance.supportsPrinting;
```

Reply:
Small bunch... but yes you're true.